config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,2 @@
+ActiveModelSerializers.config.adapter = :json_api
+ActiveModelSerializers.config.key_transform = :underscore
```

Comment:
To use https://github.com/chingor13/json_api_client, we must separate with underscore.

https://github.com/chingor13/json_api_client/blob/master/lib/json_api_client/resource.rb#L424

But hyphen is recommended, so client should handle hyphen separated members.

http://jsonapi.org/recommendations/